main.go

```diff
@@ -102,20 +104,42 @@ func maybeWrite(output *bytes.Buffer, b []byte) {
 
 func checkPath(path string) (int, error) {
 	fset := token.NewFileSet()
-	f, err := parser.ParseFile(fset, path, nil, 0)
+
+	src, err := ioutil.ReadFile(path)
 	if err != nil {
 		return 0, err
 	}
 
-	fileBytes, err := ioutil.ReadFile(path)
+	// Run goimports, which also runs gofmt.
+	importOpts := imports.Options{
```

Comment:
The thing that held me back from doing this last time is that it would be better if we could pass all options through to `goimports`, the way it does to `gofmt`. This is an interesting idea to just set defaults. Could it have problems? Do people have custom options they set? I guess probably not...

Reply:
If someone want a custom `goimports` invocation, I think it is fine that they an run that themselves and `crlfmt` can just do what we say is crl style.

maybe the `imports.Process` should be skipable, if you really only want the function formatting, but eh, I don't care too much.